src/measure_theory/borel_space.lean

```diff
@@ -244,6 +244,18 @@ begin
   exact measurable_prod_mk hf hg
 end
 
+lemma measurable_max {α β}
```

Comment:
could you name this `measurable.max`? Then, if you have `hf` and `hg` saying that `f` and `g` are measurable, you can write `hf.max hg` to say that their max is measurable. Same thing for `measurable.min` (and it would also be worth renaming `measurable_add` and `integrable_add` and so on in the same way...)

Reply:
I'm not sure how this works. Simply change the name to `measurable.max` doesn't work. Is this because `measurable.max` is in the namespace `measure_theory`?

Reply:
`measurable.max` should be placed in the same namespace as `measurable`.

Reply:
So I changed everything I can, e.g., `integrable_neg` and `integrable_smul` etc., but I'm not sure if I should move all `measurable_something` out of the namespace `measure_theory`.

Reply:
I would move `measurable` out of the namespace `measure_theory`: it is weird to have it there, and not `integrable`.